src/abstractdataframe/iteration.jl

```diff
@@ -125,7 +124,7 @@ julia> collect(eachcol(df, true))
  :x => [1, 2, 3, 4]
  :y => [11, 12, 13, 14]
 
-julia> collect(eachcol(df, false))
+julia> collect(eachcol(df))
```

Comment:
Maybe put the example with `eachcol(df, true)` at the end, since it's not the default?

Reply:
fixed